client/my-sites/plans/current-plan/header.jsx

```diff
@@ -67,7 +67,7 @@ class CurrentPlanHeader extends Component {
 				<div className={ classes }>
 					<span className="current-plan__header-expires-in">
 						{ hasAutoRenew
-							? translate( 'Set to Auto Renew on %s.', { args: currentPlan.userFacingExpiryMoment.format( 'LL' ) } )
+							? translate( 'Set to Auto Renew on %s.', { args: currentPlan.autoRenewDateMoment.format( 'LL' ) } )
 							: translate( 'Expires on %s.', { args: currentPlan.userFacingExpiryMoment.format( 'LL' ) } )
```

Comment:
> userFacingExpiry stays. I shouldnt change it on the endpoint and we shouldnt mess with it. Reverted endpoint change.

Okay. We can always remove it in another PR if needed.

Hopefully no one will confuse its name ("user facing expiry") with meaning that it's the expiry date user should see. Because it's not.

I think that's the reason this bug even exists. When @mtias/others were adding the renewal/expiration date to My Plans page, they simply assumed `userFacingExpiry` means the renewal/expiration date user should see. We (those who reviewed the original PR) made this assumption as well and we were all wrong.